src/main/java/io/vertx/rabbitmq/impl/RabbitMQClientImpl.java

```diff
@@ -776,4 +793,66 @@ public void shutdownCompleted(ShutdownSignalException cause) {
   private interface ChannelHandler<T> {
     T handle(Channel channel) throws Exception;
   }
+  
```

Comment:
can we try reusing Vert.x SSLHelper class here ?

Reply:
Yes that is possible. For that to make more sense the RabbitMQOptions should be refactored and extend NetClientOptions

Reply:
Would you agree on the above?

Reply:
I think instead you can create NetClientOptions from RabbitMQOptions that is less constraining.

Reply:
I actually went for the extending  NetClientOptions. It makes the usage more coherent with e.g. vertx-mqtt.
Also gives TCPSSLOptions for free.